src/cmd/keybaseca/keybaseca.go

```diff
@@ -126,6 +144,46 @@ func serviceAction(c *cli.Context) error {
 	return nil
 }
 
+// The action for the `keybaseca sign` subcommand
+func signAction(c *cli.Context) error {
+	// Skip validation of the config since that relies on Keybase's servers
```

Comment:
might be nicer to call the validation function anyway, and add a parameter that's something like `noKeybase`, and handle it explicitly over there.